fbpic/particles/elementary_process/ionization/inline_functions.py

```diff
@@ -31,11 +26,6 @@ def get_E_amplitude( ux, uy, uz, Ex, Ey, Ez, cBx, cBy, cBz ):
 
     return( math.sqrt( E2_on_particle ), gamma )
 
```

Comment:
Remove compilation (compilation is now done in the files that import these inline functions.)